src/views/create-update-recipe.twig

```diff
@@ -43,6 +44,12 @@
                 window.parent.zrdnAddImageHandler(zrdnRecipeImageSelected);
             });
 
+            $('#skip-button').click(function(e) {
```

Comment:
We should only set cookie on backend and not two places. We're not using ajax much at all right now and this tiny thing doesn't help. In the future, I'd love to switch everything to AJAX and a good frontend JavaScript library!